.gitignore

```diff
@@ -1,6 +1,7 @@
 # RStudio
 .Rproj.user
 **/renv/
+.Rprofile
```

Comment:
We have a couple `.Rprofiles` in the repo already, wonder if we should clean them up and remove them from `manifest.json`s.

To be clear I don't mean in this PR